### PR TITLE
Removing ose_images.sh from OSO job

### PR DESCRIPTION
**Pull request by adammhaile**
@jupierce  - Depends on https://github.com/openshift/enterprise-images/pull/143

### CODE REVIEW

File: jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh [hunk 1]
```diff
@@ -88,6 +88,20 @@ elif [ "${BUILD_MODE}" == "release" ] ; then
     SPEC_VERSION_COUNT=6
 fi
 
+echo
+echo "=========="
+echo "Setup OIT stuff"
+echo "=========="
+
+
+pushd ${BUILDPATH}
```
Comment (jupierce) [inline jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh]: no reason to push & pop that I see.

File: jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh [hunk 2]
```diff
@@ -88,6 +88,20 @@ elif [ "${BUILD_MODE}" == "release" ] ; then
...
+OIT_DIR="${BUILDPATH}/enterprise-images/"
```
Comment (jupierce) [inline jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh]: I'd prefer WORKSPACE to BUILDPATH. BUILDPATH is a 'go' directory.

File: jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh (hunk 1 above)
Comment (adammhaile) [reply jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh]: Fair... was just copy pasted from OSE. Will remove.

File: jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh (hunk 2 above)
Comment (adammhaile) [reply jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh]: Again, copy paste from OSE, but will change.

**Comment by adammhaile** [comment]
@jupierce - all notes addressed.

**Pull request merged**